src/models/event.js

```diff
@@ -784,6 +784,33 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     replacingEventId() {
         return this._replacingEvent && this._replacingEvent.getId();
     },
+
+    /**
+     * Summarise the event as JSON for debugging. If encrypted, include both the
+     * decrypted and encrypted view of the event.
+     *
+     * @return {Object}
+     */
+    toJSON() {
```

Comment:
teeeeeeeeecccchnnically this is `toObject` because JSON is a string

Reply:
Ah, I should have explained further... It has this name for use with `JSON.stringify`, which checks objects for functions named `toJSON` and will call them to customise the output if they are defined. I'll add this to the method comment.